packages/react-examples/src/react/Dropdown/docs/DropdownBestPractices.md

```diff
@@ -26,3 +26,9 @@ By default, the Dropdown truncates option text instead of wrapping to a new line
 Because this can lose meaningful information, it is recommended to adjust styles to wrap the option text.
 
 The `Dropdown with wrapping option text` example demonstrates how to override truncation styles to support wrapping. The default style will continue to truncate to support existing implementations.
+
+#### Selected Option Styles
```

Comment:
We're super inconsistent about this but the Microsoft style guide says to use sentence case
```suggestion
#### Selected option styles
```